command/agent/stats_endpoint.go

```diff
@@ -0,0 +1,62 @@
+package agent
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/hashicorp/nomad/client"
+)
+
+func (s *HTTPServer) StatsRequest(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
+	if s.agent.client == nil {
+		return nil, clientNotRunning
+	}
+
+	var tsRequest bool
```

Comment:
I don't really like this? Can we make the API:

* `?latest=true`
* `?since=timestamp`